src/main/resources/META-INF/plugin.xml

```diff
@@ -84,6 +84,11 @@
             <add-to-group group-id="EditorPopupMenu1" anchor="first"/>
             <separator/>
         </group>
+        <action id="CodeGPT.TriggerEditorPopup"
+          class="ee.carlrobert.codegpt.actions.editor.ShowEditorActionGroupAction"
+          text="Show CodeGPT Actions">
+            <keyboard-shortcut first-keystroke="ctrl shift alt m" keymap="$default"/>
```

Comment:
Can we add the same shortcut for mac os x as well?
https://github.com/carlrobertoh/CodeGPT/blob/platform/2024.1/src/main/resources/META-INF/plugin.xml#L77

Reply:
Sorry, as a Linux and Windows User I always forget about that "other" keyboard 😆 
Aren't they missing for the toggle Code Completions Action too?
https://github.com/carlrobertoh/CodeGPT/blob/8883200817f54f47b71264d450a4e68649696f25/src/main/resources/META-INF/plugin.xml#L125

https://github.com/carlrobertoh/CodeGPT/blob/8883200817f54f47b71264d450a4e68649696f25/src/main/resources/META-INF/plugin.xml#L134

Reply:
Please ignore my last comment, sorry. The `ctrl shift alt m` combination is already valid on mac as `shift option meta m`. No changes needed.

Reply:
I removed the additional keyboard-shortcuts again